content/docs/replication/deployment/install-repctl.md

```diff
@@ -7,8 +7,7 @@ description: Installation of CSM for Replication using repctl
 
 ## Install Replication Walkthrough
 
-Here are simple steps on how you can start using Container Storage Modules (CSM) for Replication with
-help from `repctl`
```

Comment:
Change line 10 to read: You can start using Container Storage Modules (CSM) for Replication with help from repctlusing these simple steps: